README.md

```diff
@@ -83,15 +91,15 @@ console.log(cryptoSocket.Exchanges['bitfinex'])
 
 ## FAQ
 
-** Does this store data? **
+### Does this store data?
 
-Nope. Ticker values (in most cases last sale price) is stored in memory.
+Nope. Ticker values (in most cases last sale price) are stored in memory.
 
-*** Why? ***
+### Why?
 
 Other popular BTC average modules require incredible amount of bandwidth at regular intervals which can make deployment difficult in many situations. Websockets are obviously the way to go for real time data, as constantly querying many exchanges can be taxing.
 
-This module **does not include exchange API's that do not have web sockets**. Why? Theres a billion other modules that do something similar, and perhaps this is a wake up call to any exchange that does not offer rhobust websocket support.
+This module **does not include exchange API's that do not have web sockets**. Why? There's a billion other modules that do something similar, and perhaps this is a wake up call to any exchange that does not offer robust websocket support.
```

Comment:
s/`There's`/`There are`

We're referring to multiple items (billions of modules).